src/Client.php

```diff
@@ -243,6 +243,14 @@ protected function getExceptionMap()
     }
 
     /**
+     * @return LoggerInterface
+     */
+    public function getLogger()
```

Comment:
Yeah, but it would be a matter of ease of use.
Maybe you're setting up the client somewhere, and using it somewhere else. This way you don't have to keep yourself the logger around, you can simply rely on the client to store it for you...